### PR TITLE
form letters: fix section names and add "from the {{ section }}"

### PR DESCRIPTION
**Pull request by cantsin**
[Link to ZenHub issue.](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/665)

## What does this change?

- Expands section names to their proper equivalents (e.g., "Disability" -> "Disability Rights")
- Migrates response templates to have a "the" before the section name (e.g., "Response: Your Civil Rights Division Report - EJX-5W2 from the Housing and Civil Enforcement Section")

### Author

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [x] Check for [accessibility](/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

File: crt_portal/cts_forms/migrations/0087_standardize_form_letter_subjects.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.2.13 on 2020-08-17 18:07
+
+from django.db import migrations
+
+
+def tweak_subject_lines(apps, schema_editor):
+    # add "the" as a prefix
+    common_subject = "Response: Your Civil Rights Division Report - {{ record_locator }} from the {{ section_name }} Section"
+    templates = ResponseTemplate.objects.filter(subject__icontains='from {{ section_name }}')
```
Comment (Jkrzy) [inline crt_portal/cts_forms/migrations/0087_standardize_form_letter_subjects.py:9]: Ooo, we need to import `ResponseTemplate` from somewhere 😃

Comment (cantsin) [reply crt_portal/cts_forms/migrations/0087_standardize_form_letter_subjects.py:9]: :see_no_evil: oh boy. my mistake completely.

**Comment by cantsin** [comment]
Subject line is getting a bit long, but not sure if there's anything we can do there.

File: crt_portal/cts_forms/migrations/0087_expand_form_subject_length.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+    dependencies = [
+        ('cts_forms', '0086_materialize_trends_view'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='report',
+            name='assigned_section',
+            field=models.TextField(choices=[('ADM', 'Administrative'), ('APP', 'Appellate'), ('CRM', 'Criminal'), ('DRS', 'Disability Rights'), ('ELS', 'Employment Litigation'), ('EOS', 'Educational Opportunities'), ('FCS', 'Federal Coordination and Compliance'), ('HCE', 'Housing and Civil Enforcement'), ('IER', 'Immigrant and Employee Rights'), ('SPL', 'Special Litigation'), ('VOT', 'Voting')], default='ADM'),
```
Comment (cantsin) [inline crt_portal/cts_forms/migrations/0087_expand_form_subject_length.py:16]: This alter field change isn't necessary and doesn't really do anything to the database (we don't change the section choice keys), but the auto-migration picked it up.

Comment (Jkrzy) [reply crt_portal/cts_forms/migrations/0087_expand_form_subject_length.py:16]: Good catch!

We should consider adding this to our CI pipeline, https://docs.djangoproject.com/en/2.2/ref/django-admin/#cmdoption-makemigrations-check

**Pull request merged**